.github/workflows/deploy-dev.yml

```diff
@@ -13,10 +13,14 @@ on:
 jobs:
   deploy:
     runs-on: ubuntu-20.04-self-hosted
+
+    env:
+      EXCLUDE_LIST: "slavakirichenko"
```

Comment:
I think you've used it for testing. We don't need it in the final version

Reply:
Fixed!